.github/workflows/main_push.yml

```diff
@@ -45,14 +45,14 @@ jobs:
       if: matrix.platform == 'windows' && steps.cache-conan.outputs.cache-hit != 'true'
       run: |
         pip install pip --upgrade
-        pip install conan==1.59.0
+        pip install conan<2.0.0
```

Comment:
they kept releasing 1.x after 2.x was released, they're at 1.61 or so now

Reply:
still loathing to invest that one hour to move any of my projects to 2.x, but I guess the time will come eventually